src/main/java/io/github/lottetreg/httpserver/Route.java

```diff
@@ -0,0 +1,66 @@
+package io.github.lottetreg.httpserver;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class Route extends BaseRoute {
+  private String controllerName;
+  private String actionName;
+  private String controllersPackage;
+
+  Route(String path, String method, String controllerName, String actionName) {
+    super(path, method);
+    this.controllerName = controllerName;
+    this.actionName = actionName;
+    this.controllersPackage = "io.github.lottetreg.httpserver.controllers";
```

Comment:
I'll probably want to keep this somewhere more configurable, but I'm not sure where yet